pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -549,6 +533,47 @@ func ReadAuthorizationModel(
 	}, nil
 }
 
+// ReadLatestAuthorizationModel reads the latest authorization model corresponding to the store
+func ReadLatestAuthorizationModel(
```

Comment:
Can we please verify this is getting served from an index and avoiding a larger scan?

If we create several tens of thousands of models and write them to our store does the OrderBy followed by Limit degrade with more models. I would suspect it does. Let's confirm/deny this.

Reply:
In a system with many stores, we see the following in Postgres

```
postgres=# explain select authorization_model_id, schema_version, type, type_definition, serialized_protobuf from authorization_model where store = '01HQBMGR6AV7H6MT4ZVK3CARR4' order by authorization_model_id desc limit 1;
                                                      QUERY PLAN
----------------------------------------------------------------------------------------------------------------------
 Limit  (cost=0.15..6.17 rows=1 width=160)
   ->  Index Scan Backward using authorization_model_pkey on authorization_model  (cost=0.15..12.18 rows=2 width=160)
         Index Cond: (store = '01HQBMGR6AV7H6MT4ZVK3CARR4'::text)
(3 rows)

```